rllib/algorithms/algorithm_config.py

```diff
@@ -1011,6 +1011,20 @@ def validate(self) -> None:
             else:
                 self.rl_module_spec = default_rl_module_spec
 
+            if self.model["custom_model"] is not None:
+                raise ValueError(
+                    "Cannot use `custom_model` option with RLModule API."
```

Comment:
You should say what they should do instead if they want to use custom_model (i.e. disabling RLModule via what API? or if they want to migrate to RLModule, migrate their custom model to a custom RLModule) Give them clear instructions.